website/docs/commands/import.html.md

```diff
@@ -110,18 +110,50 @@ specifying `-config=""` (empty string). This is useful in situations where
 you want to manually configure the provider because your configuration
 may not be valid.
 
-## Example: AWS Instance
+## Example: Import into Resource
 
-This example will import an AWS instance:
+This example will import an AWS instance into the `aws_instance` resource named `foo`:
 
 ```shell
 $ terraform import aws_instance.foo i-abcd1234
 ```
 
-## Example: Import to Module
+## Example: Import into Module
 
-The example below will import an AWS instance into a module:
+The example below will import an AWS instance into the `aws_instance` resource named `bar` into a module named `foo`:
 
 ```shell
 $ terraform import module.foo.aws_instance.bar i-abcd1234
 ```
+
+## Example: Import into Resource using count
+
+The example below will import an AWS instance into the first instance of the `aws_instance` resource named `baz` using
```

Comment:
I find the phrase "using count" (and "using for_each") somewhat confusing. I prefer something along the lines of "created with count" and "created with for_each" - what do you think?

Reply:
Opted for "configured with" since these items may not necessarily have been created in Terraform with `count`/`for_each` initially -- hope that's okay, but please let me know if you have strong opinions.